frontend/app/work_packages/services/work-package-field-service.js

```diff
@@ -393,6 +395,29 @@ module.exports = function(
     return WorkPackagesHelper.formatValue(value, mappings[field]);
   }
 
+  function submitWorkPackageChanges(notify, refreshWorkPackages) {
+    // We have to ensure that some promises are executed earlier then others
+    var fields = Object.values(EditableFieldsState.submissionPromises)
+                       .sort(function(a, b) { a.order >= b.order });
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.